boltdb/boltdb_manager_test.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package boltdb
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/google/ts-bridge/storage"
+	"github.com/timshannon/bolthold"
+)
+
+// TODO(temikus): Remove this helper in favour of T.Cleanup when we upgrade to Go 1.14+
+// See https://tip.golang.org/doc/go1.14#testing
+func CleanupStore(manager storage.Manager, records []string) error {
+	err := manager.CleanupRecords(nil, records)
+	if err != nil {
```

Comment:
If you don't mind me asking a dumb question - how do I combine lines here?

If I do `if tempFile, err :=...; err != nil {` then this will push tempFile into local scope, no?

Reply:
Github shows this comment on a different line in the file view and the commit view. Weird. Please ignore this comment for the tempFile.